code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -343,3 +343,10 @@
 		recharging_time = world.time + recharging_rate
 	else
 		to_chat(user, "<span class='warning'>Something prevents you from dashing forward!</span>")
+
+/obj/item/clothing/shoes/ducky
+	name = "rubber ducky shoes"
+	desc = "These shoes are made for quacking, and thats just what they'll do."
+	icon_state = "ducky"
+	item_state = "ducky"
+	shoe_sound = list('sound/items/squeaktoy.ogg' = 1)
```

Comment:
`shoe_sound` does not expect a list. It expects a string which is the sound.
Example
![image](https://user-images.githubusercontent.com/15887760/89191018-9a65c300-d5a2-11ea-95ed-2c80d983174a.png)

Reply:
I had trouble getting sounds using shoe_sound = squeaktoy or shoe_sound = squeaktoy.ogg, so I tried using what code I found for the rubberducky (which used a list for whatever reason). After some testing I found that using a filepath works fine, so I just commmited that.